_episodes/08-development-setup.md

```diff
@@ -53,7 +53,10 @@ Let’s get started.
 
 The ESMValTool source code is available on a public GitHub repository:
 [https://github.com/ESMValGroup/ESMValTool](https://github.com/ESMValGroup/ESMValTool).
-To obtain the code, the easiest way is to clone the repository:
+To obtain the code, there are two options:
+
+1. download the code as a ZIP file from the repository.
+2. clone the repository if you want to contribute to the ESMValTool development:
```

Comment:
At some point should we also say how they will update the code when there is a new release for example?

Reply:
A new step **4 Updating ESMValTool** is added to the lesson.